tests/integration/test_batch.py

```diff
@@ -1,20 +1,26 @@
 import time
-
-from labelbox.exceptions import ProcessingWaitTimeout
 import pytest
+
+from uuid import uuid4
 from labelbox import Dataset, Project
+from labelbox.exceptions import ProcessingWaitTimeout, MalformedQueryException, ResourceConflict, LabelboxError
 
 IMAGE_URL = "https://storage.googleapis.com/diagnostics-demo-data/coco/COCO_train2014_000000000034.jpg"
+EXTERNAL_ID = "my-image"
+
+
+def get_data_row_ids(ds: Dataset):
+    return [dr.uid for dr in list(ds.export_data_rows())]
 
 
 @pytest.fixture
 def big_dataset(dataset: Dataset):
     task = dataset.create_data_rows([
         {
             "row_data": IMAGE_URL,
-            "external_id": "my-image"
+            "external_id": EXTERNAL_ID
         },
-    ] * 2)
+    ] * 3)
```

Comment:
@vbrodsky so this was 250 before (hence big_dataset). I remember you [said](https://labelbox.slack.com/archives/C05J7G8PE4V/p1690224623159979) 250 was giving time out errors, but still, shouldn't we at least raise it to 50? Otherwise this fixture is useless.

Reply:
@attila-papai let me try to find a slack thread on it... in summary 250+ seems to be a magic number for Richard to catch some bugs so we need to sort it out... @kkim-labelbox perhaps we can add this as a story to me print? (and pull another sdk refactoring)

Reply:
2 or 3 makes no difference for large batch imho

Reply:
also see here followup discussion on 250 big dataset rows with Richard if you are so curious https://labelbox.slack.com/archives/C03F33JDRGD/p1690217836296689

Reply:
so, what should I change this number to? 50 or 250 ?

Reply:
@vbrodsky @attila-papai so, what should be the value change to?

Reply:
I recommend keep it to 2. I am working on a story to optimize sdk tests and have in mind of it to either bring back 250 or figure what to do to make sure we test this feature and Richard is happy :)